test/parallel/test-process-env.js

```diff
@@ -67,3 +67,17 @@ assert.equal(3, date.getUTCHours());
 assert.equal(5, date.getHours());
 */
 /* eslint-enable max-len */
+
+// Environment variables should be case-insensitive on Windows, and
+// case-sensitive on other platforms.
+process.env.TEST = 'test';
+assert.strictEqual(process.env.TEST, 'test');
+// Check both mixed case and lower case, to avoid any regressions that might
+// simply convert input to lower case.
+if (common.isWindows) {
+  assert.strictEqual(process.env.test, 'test');
+  assert.strictEqual(process.env.teST, 'test');
+} else {
+  assert.strictEqual(undefined, process.env.test);
```

Comment:
Nit: can you please swap the arguments? The first one is `actual` and second one `expected`.